StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1500CurlyBracketsForMultiLineStatementsMustNotShareLine.cs

```diff
@@ -204,6 +204,8 @@ private static void CheckCurlyBracketToken(SyntaxNodeAnalysisContext context, Sy
                 case SyntaxKind.CloseParenToken:
                 case SyntaxKind.CommaToken:
                 case SyntaxKind.SemicolonToken:
+                case SyntaxKind.DotToken:
+                case SyntaxKind.QuestionToken:
```

Comment:
:exclamation: I don't think we need to include `QuestionToken`. An object creation expression will never result in `null`, so the null propagation operator would be pointless.